corehq/ex-submodules/pillowtop/processors/elastic.py

```diff
@@ -72,15 +72,27 @@ def process_change(self, change):
             return
 
         if change.deleted and change.id:
-            logger.info(f'Attempting to delete document for change {change.id}')
             doc = change.get_document()
             if doc and doc.get('doc_type'):
+                logger.info(
```

Comment:
Aren't these the same? The change `id` is the id of the document, and is used by `get_document`.

Reply:
You're absolutely correct 🤦🏻